packages/mdx/index.js

```diff
@@ -103,7 +103,12 @@ function sync(mdx, options) {
   const opts = Object.assign({}, DEFAULT_OPTIONS, options)
   const compiler = createCompiler(opts)
 
-  const { contents } = compiler.processSync(mdx)
+  const fileOpts = { contents: mdx };
```

Comment:
This looks amazing! You *could* pass {contents: mdx, path: opts.filepath}, but that's just nitpicking.

Looks great, lekker bezig!

Reply:
Hmm yes, I didn't do it in this case since the mdx transpiler didn't work for me when `path: undefined`. I can still change it since that's probably something that won't often happen, but this one is more correct methinks.

Also, dankjewel :D! Leuk om zoveel Nederlanders hier te zien!